utils/parsers.py

```diff
@@ -425,3 +425,33 @@ class SupportAPIParser(PrimeAPIParser):
     """ Parser class for the Support API. """
 
     api_file = "https://raw.githubusercontent.com/transcom/mymove/master/swagger/support.yaml"
+
+    def _custom_request_validation(self, path, method, request_data):
+        """
+        Custom post-data generation validation for the Support API. Note that request_data is mutable and directly
+        modified, and therefore doesn't need to be returned.
+        """
+        if path == "/move-task-orders" and method == "post":
+
+            move_task_order = request_data
+            move_orders = request_data["moveOrder"]
+            customer = move_orders["customer"]
+            entitlement = move_orders["entitlement"]
```

Comment:
Splitting it out this way makes it really clear later on! I like it. There are so many nested objects in this payload, oof

Reply:
Yes, a lot. It's a tough one.